roadmap.md

```diff
@@ -0,0 +1,11 @@
+# Roadmap for v0.6
+
+1. CUJ.The CUJ is to track model development in Notebook and Kubeflow Pipeline. ([deck](http://bit.ly/kf-meta-cuj))
+
+2. API and metadata specifications. The goal is to define API and a set of built-in metadata specifications that can support the CUJ above; The API should be able to handle customized metadata specifications.
+
+3. Implementations. We plan to leverage [MLMD](https://github.com/google/ml-metadata) for an implementation of the API, because it is already used by Kubeflow Pipeline. Different implementations are welcome and should be easily discoverable.
+
+4. Python library. We plan to create a Python library for logging metadata in Notebook and Kubeflow Pipeline.
```

Comment:
Is the logged metadata mutable? Can the mutability be controlled by the user? If so, we may need all CRUD for the logged metadata.

Reply:
Yes, in general metadata is mutable and CRUD API will be provided. But some field are immutable, such as the uuid, type.

Reply:
Sounds good.